<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1410</width>
    <height>874</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="example">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>1341</width>
      <height>551</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="list_button">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>809</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>ОТВЕТЫ И РЕШЕНИЯ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="test_button">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>809</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>ТЕСТ</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="right_number">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>400</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>350</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество правильных ответов</string>
    </property>
   </widget>
   <widget class="QPushButton" name="answer_button">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>730</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Ответить</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="list_buttons">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>501</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="number1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Задание 1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="number2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Задание 2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="number3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>0</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Задание 3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="number4">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>0</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Задание 4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="number5">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>0</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Задание 5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="number6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Задание 6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="number7">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Задание 7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="number8">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Задание 8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="number9">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>50</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Задание 9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="number10">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>50</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Задание 10</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="next_button">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>730</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Седущее задание</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>740</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Предыдущее задание</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_button">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>730</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Показать решение и ответ</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1410</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
